data/pb/ACCESSCONTROL-ACTIONS-TEST/actions-test.json

```diff
@@ -15982,6 +15982,37 @@
       "leftIcon": "custom:edcr",
       "rightIcon": "",
       "queryParams": ""
+    },
+    {
+      "id": 1940,
+      "name": "rainmaker-common-wns",
+      "url": "card",
+      "displayName": "SURE Dashboard",
+      "orderNumber": 2,
+      "parentModule": "",
+      "enabled": true,
+      "serviceCode": "integration",
+      "code": "",
+      "path": "",
+      "navigationURL": "integration/dss/home",
```

Comment:
This is added twice. Please check `integration/dss/home`

Reply:
Even on UAT it is already there.